app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (-1, -1, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (100, 100, [21, 17]),
+    ]
+)
+def test_get_human_age(cat_age: int, dog_age: int, expected: list) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

Comment:
add tests for this case 
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/93856423/f02a86c4-b4c4-4c53-be6e-50c8a7f4b332)